lib/concurrent/edge/future.rb

```diff
@@ -775,14 +775,14 @@ def call_callback(method, state, *args)
         self.send method, state, *args
       end
 
-      def pr_async_callback_on_success(state, executor, callback)
-        pr_with_async(executor, state, callback) do |state, callback|
+      def pr_async_callback_on_success(_state, _executor, _callback)
+        pr_with_async(_executor, _state, _callback) do |state, callback|
           pr_callback_on_success state, callback
```

Comment:
I think these names suggests that the `_state, _executor, _callback` is unused, could you do `pr_async_callback_on_success(state, executor, callback)` and `pr_with_async(executor, state, callback) { |st, cb| pr_callback_on_success st, cb }` instead?